tests/unit/api/bucketGetObjectLock.js

```diff
@@ -20,7 +20,7 @@ const testBucketPutReqWithObjLock = {
     bucketName,
     headers: {
         'host': `${bucketName}.s3.amazonaws.com`,
-        'x-amz-bucket-object-lock-enabled': true,
+        'x-amz-bucket-object-lock-enabled': 'true',
```

Comment:
does not need to be this one, but can you set one of the `true` values to different cases like `True`?

Reply:
good idea, just updated while I was on it

Reply:
As a partially unrelated note, header keys should also be case insensitive such that the key `X-AMZ-bucket-object-lock-ENABLED` should also still work. Spec is here https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

Reply:
Node.js takes care of the casing and we can rely on it being lowercased https://nodejs.org/dist/latest-v10.x/docs/api/http.html#http_message_headers

Reply:
ok gotcha, thats good. Not all tools do that for you. Also, not everyone knows that headers are handled that way.